ad2games-theme.gemspec

```diff
@@ -13,6 +13,5 @@ Gem::Specification.new do |s|
   s.files     = `git ls-files`.split("\n")
   s.require_paths = ['lib', 'assets', 'fonts']
 
-  s.add_dependency 'rails', '~> 4.2.0'
```

Comment:
My main point is rather to specify the `rails` dependency at all(know I see it's actually `railties`), and `>=3` is just for a company :) Anyway, it will also work without it, so 👍